test/Concurrency/isolated_parameters.swift

```diff
@@ -59,7 +64,8 @@ func globalFuncIsolated(_: isolated A) { // expected-note{{calls to global funct
 }
 
 @available(SwiftStdlib 5.1, *)
-func testIsolatedParamCalls(a: isolated A, b: A) {
+// expected-warning@+1 {{global function with 'isolated' parameter cannot have a global actor; this is an error in Swift 6}}{{1-12=}}
```

Comment:
wording perhaps can be improved? It's a bit weird to say a function "has" an actor (though it is understandable what it means ofc still).

What do we call those things? Is `cannot have global actor annotation` more correct? 🤔

Reply:
Yeah I agree, it's awkward wording, but is definitely very short and sweet.I basically copied the wording from several other diagnostics (https://github.com/apple/swift/blob/44e6cb1d12edcc35affb79d5927b765c95ae8ca1/include/swift/AST/DiagnosticsSema.def#L4996). So for now I'm going to go with this until someone changes them wholesale.